gradio/components/video.py

```diff
@@ -202,9 +202,15 @@ def preprocess(
             video.get("is_file", False),
         )
 
+        # if client_utils.is_http_url_like(file_name):
+
         if is_file:
             assert file_name is not None, "Received file data without a file name."
-            file_name = Path(self.make_temp_copy_if_needed(file_name))
+            if client_utils.probe_url(file_name):
```

Comment:
I think its sufficient (and faster) to just do:
```suggestion
            if client_utils.is_http_url_like(file_name):
```

Reply:
Good point - pushed up!